airbyte-integrations/connectors/source-mysql-v2/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlSourceConfigurationJsonObject.kt

```diff
@@ -80,12 +79,11 @@ class MysqlSourceConfigurationJsonObject : ConfigurationJsonObjectBase() {
     @JsonSchemaInject(json = """{"order":5,"always_show":true,"airbyte_secret":true}""")
     var password: String? = null
 
-    @JsonProperty("schemas")
-    @JsonSchemaTitle("Schemas")
-    @JsonSchemaArrayWithUniqueItems("schemas")
-    @JsonPropertyDescription("The list of schemas to sync from. Defaults to user. Case sensitive.")
-    @JsonSchemaInject(json = """{"order":6,"always_show":true,"uniqueItems":true}""")
-    var schemas: List<String>? = null
+    @JsonProperty("database")
+    @JsonSchemaTitle("Database")
+    @JsonPropertyDescription("The database name.")
+    @JsonSchemaInject(json = """{"order":6,"always_show":true}""")
+    lateinit var database: String
```

Comment:
I replaced the `schemas` field, which was copy-pasted from oracle, with the `database` defintion I found in `source-mysql`. This change creates a bunch of noise in this PR.